translations/bnpchs-final-2018-tikus-atom.html

```diff
@@ -0,0 +1,95 @@
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Jelix Fingga has a house in coordinate $$$(0, 0, 0)$$$. The field near his house is resided by $$$N$$$ moles. Unfortunately, 
+  the neighbour of Jelix arm each mole with a hydrogen bomb. The bomb will explode if there is another bomb so that 
+  their distances is <b>strictly less</b> than $$$R$$$ meters.
```

Comment:
```suggestion
  Jelix's neighbour arm each mole with a hydrogen bomb. The bomb will explode immediately if there is another bomb 
  with distance <b>less than</b> $$$R$$$ meters.
```
dari baca deskripsi sisanya, kayanya ga strictly